ci-operator/jobs/stolostron/policy-collection/stolostron-policy-collection-main-periodics.yaml

```diff
@@ -17,6 +17,17 @@ periodics:
     job-release: "4.12"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-stolostron-policy-collection-main-opp-ocp4.12-interop-openshift-plus-interop-aws
+  reporter_config:
+    slack:
+      channel: '#opp-discussion'
+      job_states_to_report:
+      - success
+      - failure
+      - error
+      report_template: '{{if eq .Status.State "success"}} :slack-green: Job *{{.Spec.Job}}*
+        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :failed:
+        Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
+        logs> {{end}}'  
```

Comment:
There's some whitespace that the automation doesn't like at the end of this line

Reply:
Good catch, I updated and it resolved this failure